dev-4/YandexTests/YandexTests/Tests/DemoSeleniumEasyTest/JavaScriptAlertTest.cs

```diff
@@ -0,0 +1,114 @@
+        {
+            javaScriptAlertPage = new JavaScriptAlertPage(Driver);
+            javaScriptAlertPage.GoTo();
+        }
+
+        [Test]
+        public void TestJavaScriptConfirmBoxAcceptTest()
+        {
+            string expectedAlertText = "You pressed OK!";
+
+            // 1. Click on Click me! button
+            IWebElement element = javaScriptAlertPage.WaitingForElement(javaScriptAlertPage.clickButtonForConfirmBox);
+            element.Click();
+
+            try
+            {
+                IAlert alert = Driver.SwitchTo().Alert();
+                alert.Accept();
+                string message = javaScriptAlertPage.WaitingForElement(javaScriptAlertPage.confirmInput).Text;
+
+                Assert.AreEqual(expectedAlertText, message, "Dont displays correct message on Accept!");
+            }
+            catch(NoAlertPresentException e)
+            {
+                Assert.Pass(e.StackTrace);
+            }
+        }
+
+        [Test]
+        public void TestJavaScriptConfirmBoxCancelTest()
+        {
+            string expectedAlertText = "You pressed Cancel!";
+
+            // 1. Click on Click me! button
+            IWebElement element = javaScriptAlertPage.WaitingForElement(javaScriptAlertPage.clickButtonForConfirmBox);
+            element.Click();
+
+            try
+            {
+                IAlert alert = Driver.SwitchTo().Alert();
+                alert.Dismiss();
+                string message = javaScriptAlertPage.WaitingForElement(javaScriptAlertPage.confirmInput).Text;
+
+                Assert.AreEqual(expectedAlertText, message, "Dont displays correct message on Accept!");
+            }
+            catch (NoAlertPresentException e)
+            {
+                Assert.Pass(e.StackTrace);
+            }
+        }
+
+        [Test]
+        public void TestJavaScriptAlertBoxTest()
+        {
+            string expectedAlertText = "I am an alert box!";
+
+            // 1. Click on Click me! button
+            IWebElement element = javaScriptAlertPage.WaitingForElement(javaScriptAlertPage.clickButtonForAlertBox);
+            element.Click();
+
+            try
+            {
+                IAlert alert = Driver.SwitchTo().Alert();
+                string message = alert.Text;
+                alert.Accept();
+
+                Assert.AreEqual(expectedAlertText, message, "Dont displays correct message on Accept!");
+            }
+            catch (NoAlertPresentException e)
+            {
+                Assert.Pass(e.StackTrace);
+            }
+        }
+
+        [Test]
+        public void TestJavaScriptAlertBoxPromptTest()
+        {
+            string data = "Vladislav";
+            string expectedAlertText = $"You have entered '{data}' !";
+
+
+            // 1. Click for Promt Box button
+            IWebElement element = javaScriptAlertPage.WaitingForElement(javaScriptAlertPage.clickForPromptBoxButton);
+            element.Click();
+
+            try
+            {
```

Comment:
Don't use try/catch within your tests!!! Change it, please.
P.S. It's bad practice to handle exceptions at your tests code level. Your tests should consist of AAA pattern - Arrange, Act, Assert.